LayoutTests/imported/w3c/web-platform-tests/fetch/api/headers/headers-basic.any-expected.txt

```diff
@@ -18,4 +18,8 @@ PASS Check values method
 PASS Check entries method
 PASS Check Symbol.iterator method
 PASS Check forEach method
+PASS Iteration skips elements removed while iterating
```

Comment:
How are Gecko and Blink doing on those new checks?

Also, is this behavior specified anywhere? It is not obvious to me that the new behavior is better so I am curious.

Reply:
Right, as pointed by @annevk, we might want to check other iterators.
Maybe we should make it more explicit in how our iterators work as well.